src/main/java/frc/robot/subsystems/Rollergripper.java

```diff
@@ -14,24 +14,24 @@
 import static frc.robot.Constants.RollerGripperConstants.*;
 
 public class Rollergripper extends SubsystemBase {
-  private final CANSparkMax rightRoller = new CANSparkMax(RIGHT_ROLLER_MOTOR_ID, kBrushless);
-  private final CANSparkMax leftRoller = new CANSparkMax(LEFT_ROLLER_MOTOR_ID, kBrushless);
+  private final CANSparkMax FollowRoller = new CANSparkMax(RIGHT_ROLLER_MOTOR_ID, kBrushless);
+  private final CANSparkMax LeadRoller = new CANSparkMax(LEFT_ROLLER_MOTOR_ID, kBrushless);
 
   private final DigitalInput beambreak = new DigitalInput(INTAKE_BEAMBREAK);
 
   public final Trigger beambreakTrigger = new Trigger(() -> !beambreak.get());
 
   public Rollergripper() {
-    rightRoller.restoreFactoryDefaults();
-    rightRoller.clearFaults();
-    rightRoller.setIdleMode(CANSparkMax.IdleMode.kBrake);
+    FollowRoller.restoreFactoryDefaults();
+    FollowRoller.clearFaults();
+    FollowRoller.setIdleMode(CANSparkMax.IdleMode.kBrake);
+FollowRoller.follow(LeadRoller,true);
+    LeadRoller.restoreFactoryDefaults();
+    LeadRoller.clearFaults();
+    LeadRoller.setIdleMode(CANSparkMax.IdleMode.kBrake);
 
-    leftRoller.restoreFactoryDefaults();
-    leftRoller.clearFaults();
-    leftRoller.setIdleMode(CANSparkMax.IdleMode.kBrake);
-
-    leftRoller.setInverted(false);
-    rightRoller.setInverted(true);
+    LeadRoller.setInverted(false);
+    FollowRoller.setInverted(true);
```

Comment:
delete the left roller line (it's redundant), and set the follower to be inverted when you set it to follow the leader

Reply:
oh that's the code from 2hours ago probably